### DISCUSSION TITLE
Infra  setup RC FE and BE in GCP

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-RC/community#295] [Issues]
Hi Team,

CSP : GCP
Need help on setting up RC FE&& BE in GCP

**Comment by tejash-jl** [comment]
https://docs.sunbirdrc.dev/developer-documentation/installation-guide

**Reply by tsprasath** [reply]
@tejash-jl  I would like to setup a production environment for Sunbird-RC.Please let me know above document will works

**Reply by tejash-jl** [reply]
If you are planning to run on K8, we have the basic infra files here,
https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/infra
